app/src/utils/apiHelpers.test.tsx

```diff
@@ -0,0 +1,212 @@
+        divisions: [
+          {
+            name: "Billy Rose Theatre Division",
+            slug: "billy-rose-theatre-division",
+            collections: [],
+          },
+          {
+            name: "Carl H. Pforzheimer Collection of Shelley and His Circle",
+            slug: "carl-h-pforzheimer-collection-of-shelley-and-his-circle",
+            collections: [],
+          },
+        ],
+      })
+    );
+    const result = await getDivisionData({
+      pageNum: 1,
+      perPage: 2,
+    });
+    expect(result.divisions.length).toEqual(2);
+    expect(result).toHaveProperty("summary");
+  });
+});
+
+describe("getNumDigitizedItems", () => {
+  it("returns the correct numDigitizedItems", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        count: {
+          $: 78,
+        },
+      })
+    );
+    const result = await getNumDigitizedItems();
+
+    expect(result).toEqual("78");
+  });
+
+  it("returns the fallback numDigitizedItems on empty response", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(Promise.resolve({}));
+
+    const result = await getNumDigitizedItems();
+
+    expect(result).toEqual(
+      defaultFeaturedItem["development"].numberOfDigitizedItems
+    );
+  });
+});
+
+describe("getItemsCountFromUUIDs", () => {
+  it("should return the correct numItems for each UUID", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        nyplAPI: {
+          response: {
+            counts: {
+              count: [
+                { uuid: { $: "uuid1" }, count_value: { $: "10" } },
+                { uuid: { $: "uuid2" }, count_value: { $: "45" } },
+              ],
+            },
+          },
+        },
+      })
+    );
+
+    const uuids = ["uuid1", "uuid2"];
+
+    const results = await getItemsCountFromUUIDs(uuids);
+
+    expect(results).toEqual({
+      uuid1: "10",
+      uuid2: "45",
+    });
+  });
+
+  it("should handle missing count field gracefully", async () => {
+    const uuids = ["uuid1", "uuid2", "uuid3"];
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        nyplAPI: {
+          response: {
+            counts: {
+              count: [
+                { uuid: { $: "uuid1" }, count_value: { $: "10" } },
+                { uuid: { $: "uuid3" }, count_value: { $: "60" } },
+              ],
+            },
+          },
+        },
+      })
+    );
+    const results = await getItemsCountFromUUIDs(uuids);
+    expect(results).toEqual({
+      uuid1: "10",
+      uuid3: "60",
+    });
+  });
+});
+
+describe("getRandomFeaturedItem", () => {
+  it("returns expected item", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve(mockFeaturedItemResponse)
+    );
+    const item = await getRandomFeaturedItem();
+    expect(item).toEqual(mockFeaturedItemResponse);
+    expect(item).toHaveProperty("capture");
```

Comment:
Similar comment as above, add an expectation for the URL that is called. I think that's a good test.